packages/neos-ui-editors/src/EditorEnvelope/index.js

```diff
@@ -86,7 +102,9 @@ export default class EditorEnvelope extends PureComponent {
 
         return (
             <div>
-                {this.renderLabel()}
+                <span title={this.props.helpMessage}>
```

Comment:
Maybe a fallback to the label if no helpmessage is set?

Reply:
If there is no helpmessage there should be no helpmessage icon rendered, so there also shouldn't be a fallback IMO